openlane/steps/odb.py

```diff
@@ -477,32 +481,61 @@ def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
 
 
 @Step.factory.register()
-class HeuristicDiodeInsertion(OdbpyStep):
+class DiodesOnPorts(CompositeStep):
```

Comment:
Need to document somewhere that old DiodeOnPorts and new DiodeOnPorts have different behaviors. In fact, this maybe considered somewhat of an API break.